src/app/page.tsx

```diff
@@ -1,9 +1,11 @@
 import Navbar from "@components/Navbar";
+import Vertical_Navbar_Component from "@components/vertical_navbar";
 
 export default function Home() {
   return (
     <main>
       <h1>Volunteer Management System Home Page</h1>
+      <Vertical_Navbar_Component></Vertical_Navbar_Component>
```

Comment:
For components that don't wrap any other ones, you can just do this tag <Vertical_Navbar_Component />

Reply:
I understand!